src/common_main.c

```diff
@@ -67,21 +67,43 @@ static bool _setup_wally(void)
     return wally_set_operations(&_ops) == WALLY_OK;
 }
 
+// Go into bootloader if there was an error during startup, so a firmware update can be
+// applied. Otherwise, if there is an Abort() during startup, there would no way to reboot into the
+// bootloader and the device would be bricked.
+static void _bootloader_autoenter(void)
+{
+    auto_enter_t auto_enter = {
+        .value = sectrue_u8,
+    };
+    upside_down_t upside_down = {
+        .value = screen_is_upside_down(),
```

Comment:
For bitboxbase it's always set to `false`. I know it doesn't really matter but it just feels inconsistent with the rest of auto-enter usage.

Reply:
True on one hand, and on the other hand it feels bad to add ifdefs here :thinking: